exe/Wrapper.hs

```diff
@@ -114,13 +114,21 @@ launchHaskellLanguageServer parsedArgs = do
       callProcess e args
 #else
       let Cradle { cradleOptsProg = CradleAction { runGhcCmd } } = cradle
-      (CradleSuccess ghcBinary) <- fmap trim <$> runGhcCmd ["-v0", "-package-env=-", "-e", "putStr =<< System.Environment.getExecutablePath"]
-      (CradleSuccess libdir) <- HieBios.getRuntimeGhcLibDir cradle
+      ghcBinary <- (fmap trim <$> runGhcCmd ["-v0", "-package-env=-", "-e", "do e <- System.Environment.getExecutablePath ; System.IO.putStr e"])
```

Comment:
Worth a comment, very non-obvious that you're doing it this way to be compatible with `NoImplicitPrelude`.

Also, this probably won't work with `RebindableSyntax` still, but surely nobody sets *that* globally...